app/views/advanced_search_finder/show.html.erb

```diff
@@ -9,48 +9,52 @@
 <% end %>
 
 <%= render "govuk_publishing_components/components/inverse_header", { full_width: true } do %>
-  <div class="full-page-width-wrapper">
-    <div class="breadcrumb-wrapper">
-      <%= render 'govuk_publishing_components/components/breadcrumbs', finder.breadcrumbs.merge(inverse: true) %>
+  <div class="govuk-width-container">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <%= render 'govuk_publishing_components/components/breadcrumbs', finder.breadcrumbs.merge(inverse: true) %>
+      </div>
     </div>
 
-    <div class="title-and-metadata">
```

Comment:
You've removed the `title-and-metadata` class, but it's still called in a css file (_advanced-search.scss)